composer.json

```diff
@@ -50,8 +50,8 @@
         "composer-exit-on-patch-failure": true,
         "patches": {
             "drupal/core": {
-                "https://www.drupal.org/project/drupal/issues/2943172": "https://www.drupal.org/files/issues/2018-04-15/2943172-kernel-test-base-2.patch",
-                "https://www.drupal.org/project/drupal/issues/2599228": "https://www.drupal.org/files/issues/2599228-31-tests-only.patch"
+                "https://www.drupal.org/project/drupal/issues/2943172": "https://www.drupal.org/files/issues/2018-07-05/2943172-kernel-test-base-3.patch",
+                "https://www.drupal.org/project/drupal/issues/2599228": "https://www.drupal.org/files/issues/2018-05-17/2599228-51.patch"
```

Comment:
This is changing patches which can cause errors in other components that depend on oe_multilingual. In case composer-patches encounters two different versions of the same patch it will throw an error. This will happen for example in oe_theme which depends on oe_multilingual.

I think it is best to wait with this until we were able to remove all the patches.

Reply:
Since we are going to deal with the patches in a follow up I'm approving this PR.